t/re-find.t

```diff
@@ -32,6 +32,8 @@ our $HttpConfig = <<_EOC_;
         -- jit.opt.start("hotloop=1")
         -- jit.opt.start("loopunroll=1000000")
         -- jit.off()
+
+        ngx.re.opt("jit_stack_size", 128 * 1024)
```

Comment:
I think we should add a new test file like `re-opt.t` for this new feature instead.